state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -400,26 +402,21 @@ pub mod pallet {
 						});
 						// Handle swap completion logic.
 						match &swap.swap_type {
-							SwapType::Swap(destination_address) =>
-								if !egress_amount.is_zero() {
-									let egress_id = T::EgressHandler::schedule_egress(
-										swap.to,
-										egress_amount,
-										destination_address.clone(),
-										None,
-									);
-
-									Self::deposit_event(Event::<T>::SwapEgressScheduled {
-										swap_id: swap.swap_id,
-										egress_id,
-										asset: swap.to,
-										amount: egress_amount,
-									});
-								} else {
-									Self::deposit_event(Event::<T>::EgressAmountZero {
-										swap_id: swap.swap_id,
-									})
-								},
+							SwapType::Swap(destination_address) => {
+								let egress_id = T::EgressHandler::schedule_egress(
+									swap.to,
+									egress_amount,
+									destination_address.clone(),
+									None,
+								);
+
+								Self::deposit_event(Event::<T>::SwapEgressScheduled {
+									swap_id: swap.swap_id,
+									egress_id,
+									asset: swap.to,
+									amount: egress_amount,
```

Comment:
I noticed this is the amount including fees - not what is actually received by the user. This should probably be fixed too.

Reply:
Would double check this with product first - there was some discussion about fees included or not included in these amounts

Reply:
as far as i see, we dont use the `amount` of the `SwapEgressScheduled` event at the moment, so changing it would be fine on our side

Reply:
Saying we have scheduled an egress for X amount if the actual amount is (X - fee) seems wrong? I'd rather we expose the correct value. Less confusing.